Cargo.toml

```diff
@@ -75,7 +74,7 @@ ndarray = ["dep:ndarray"]
 proptest = ["dep:proptest"]
 rand = ["dep:rand"]
 rkyv = ["dep:rkyv"]
-rkyv-safe = ["dep:bytecheck", "rkyv/validation"]
+rkyv-safe = ["rkyv/validation"]
```

Comment:
Ideally this should be renamed to `rkyv-validation` to mirror the feature flag name upstream but that is a breaking change or there would have to be a `#[cfg(any("rkyv-safe", "rkyv-validation"))]` interim approach.

Reply:
This is a good idea. While we need to keep `rkyv-safe` for backwards compatibility, it'd be easy to introduce a new feature with this name, particularly if it makes discoverability easier..